pkg/constants/constants.go

```diff
@@ -295,9 +299,9 @@ const (
 	// InstallLogsAWSS3BucketEnvVar is the environment variable specifying the S3 bucket to use.
 	InstallLogsAWSS3BucketEnvVar = "HIVE_INSTALL_LOGS_AWS_S3_BUCKET"
 
-	// HiveSyncSetsFakeApplyEnvVar is set to true by the hive-operator when a developer has configured HiveConfig to
-	// fake out all apply operations for simulated scale testing.
-	HiveSyncSetsFakeApplyEnvVar = "HIVE_SYNCSETS_FAKE_APPLY"
+	// HiveFakeClusterAnnotation can be set to true on a cluster deployment to create a fake cluster that will
+	// launch an install job that never provisions resources, or tries to communicate with the cluster.
```

Comment:
> that never provisions resources

it's at least clear to that this means that the results will be positive i.e. the object will say installed but actuall no cluster was created..

Reply:
Updated.